requirements-dev.txt

```
@@ -16,7 +16,7 @@ codespell==2.2.5
 colorama==0.4.6
 commonmark==0.9.1
 coverage==6.5.0
-git+https://github.com/canonical/craft-application.git@main#egg=craft_application
+git+https://github.com/canonical/craft-application.git@fix-metadata-version#egg=craft_application
```

```suggestion
git+https://github.com/canonical/craft-application.git@main#egg=craft_application
```

Just since it's merged :-)

---

rebased this and removed the commit that adds this